ios/Podfile

```diff
@@ -10,32 +10,78 @@ project 'Runner', {
   'Release' => :release,
 }
 
-def flutter_root
-  generated_xcode_build_settings_path = File.expand_path(File.join('..', 'Flutter', 'Generated.xcconfig'), __FILE__)
-  unless File.exist?(generated_xcode_build_settings_path)
-    raise "#{generated_xcode_build_settings_path} must exist. If you're running pod install manually, make sure flutter pub get is executed first"
+def parse_KV_file(file, separator='=')
```

Comment:
This happened to my Podfile when I rebased, deleted, and regenerated. 😬  
Do these changes work on your end?

Reply:
I think if it is just auto-generated and the application runs well we should be fine.